cpp/src/slice2cs/Gen.cpp

```diff
@@ -1900,6 +1931,7 @@ Slice::Gen::TypesVisitor::visitEnum(const EnumPtr& p)
     emitDeprecate(p, 0, _out, "type");
     emitCommonAttributes();
     emitCustomAttributes(p);
+    writeTypeDocComment(p, "");
```

Comment:
What does `""` mean here?

Reply:
Fixed, it's the deprecated reason if the type was deprecated.